pkg/util/utils.go

```diff
@@ -1,42 +1,55 @@
 package util
 
 import (
-	"fmt"
 	"math/rand"
 	"os"
 	"strings"
 	"time"
 
-	"log"
 	"path"
 
-	"github.com/fatih/color"
 	gohomedir "github.com/mitchellh/go-homedir"
+	log "github.com/sirupsen/logrus"
 )
 
 func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
-// Failed will print an red error message and exit with failure.
+// Failed will print a red error message and exit with failure.
 func Failed(format string, a ...interface{}) {
-	Error(format, a...)
-	os.Exit(1)
+	if a != nil {
```

Comment:
not sure i understand the reason for condition check here

Reply:
IIRc, log.Fatalf("some format string", nil) makes ugly output, pointing out the nil to the end user and calling it bad.